tests/integration/test_hitl_loop_resume_simple.py

```
@@ -0,0 +1,301 @@
+    assert call_count[0] == 1, f"Agent should be called once before pause, got {call_count[0]}"
+    
+    # Verify agent output is captured in context
+    ctx1 = result1.final_pipeline_context
+    if hasattr(ctx1, "steps") and "agent" in ctx1.steps:
+        agent_output = ctx1.steps["agent"]
+        assert agent_output is not None, "Agent output should be captured"
+        assert isinstance(agent_output, dict), "Agent output should be a dict"
+        assert agent_output.get("count") == 1, f"Agent output should show call 1, got: {agent_output}"
+    
+    # Resume
+    result2 = await runner.resume_async(result1, "yes")
```

Ruff (0.13.3)

Local variable `result2` is assigned to but never used

Remove assignment to unused variable `result2`

(F841)